ols-1.md

```diff
@@ -43,55 +43,15 @@ photos:
 {% endfor %}
 {% assign p-speakers = all-speakers | remove_first: ', ' | split: ", " | uniq | sort %}
 
-# The OLS-1 program
```

Comment:
Why did you remove this content here? I think it should stay here. Same for the other cohort files

Reply:
I moved it to the general cohort page but i see you want me to return it so i will.